transformations/transformations.js

```diff
@@ -7,6 +7,9 @@ const transformation = {
   hero_id: ({ field }) => constants.heroes[field],
   radiant_win: ({ field, match }) => {
     const isRadiantResult = isRadiant(match.player_slot);
+    if (field === undefined) {
+      return 'N/A';
```

Comment:
Maybe "N" with gray text.

Then W with green, and L with red, like how we currently do it.